cloudml-template/examples/babyweight-custom/trainer/task.py

```diff
@@ -93,7 +93,7 @@ def initialise_hyper_params(args_parser):
         If both --train-size and --num-epochs are specified,
         --train-steps will be: (train-size/train-batch-size) * num-epochs.\
         """,
-        default=10,
+        default=None,
```

Comment:
This is fine, but please consider further simplifying to allow just one appropriate way of specifying train_steps.  I generally find train_steps and train_batch_size to be more informative when compared to the often arbitrary epochs.

Reply:
@dizcology - Thank you for the comment. I tried to give two options for the user the control the total number of iterations:
1. via explicitly specifying **train_steps** (in which case, num_epochs is irrelevant). This option is probably more relevant to large training data size with unknown number of examples

2. via specifying both **num_epochs** and **train_size** (in which case, the train_steps will be (train-size/train-batch-size) * num-epochs. This option is probably more relevant to relatively small training data size with known number of examples (train_size)

However, if you think it is better to only have one way, happy to do this in the following PR :-)